### PR TITLE
Create feature to add income

### PR DESCRIPTION
**Pull request by penguin-s**

### CODE REVIEW

File: src/main/java/seedu/moneygowhere/common/Messages.java [hunk 1]
```diff
@@ -50,5 +50,14 @@ public class Messages {
             + "File loaded successfully :)";
     public static final String LOCAL_STORAGE_ERROR_SAVE_DATA = ""
             + "There is an error in saving file...\n"
-            + "File does not exist or Path is wrong :(";
+            + "File does not exist or Path is wrong :("
+            + "SYNTAX: View-Expense [-i EXPENSE_NUMBER]";
```
Comment (xzynos) [inline src/main/java/seedu/moneygowhere/common/Messages.java:54]: I believe that this string isn't part of the LOCAL_STORAGE_ERROR_SAVE_DATA variable. Could you remove it please?

File: src/main/java/seedu/moneygowhere/parser/ConsoleParser.java [hunk 2]
```diff
@@ -391,14 +471,16 @@ private static ConsoleCommandSortExpense parseCommandSortExpense(String argument
      * @throws ConsoleParserCommandDeleteExpenseInvalidException   If the command delete-expense is invalid.
      * @throws ConsoleParserCommandEditExpenseInvalidException     If the command edit-expense is invalid.
      * @throws ConsoleParserCommandSortExpenseInvalidTypeException If the command sort-expense is invalid.
+     * @throws ConsoleParserCommandAddIncomeInvalidException       If the command add-income is invalid.
```
Comment (xzynos) [inline src/main/java/seedu/moneygowhere/parser/ConsoleParser.java:474]: Thanks for updating the Javadoc comments!

**Review by xzynos — CHANGES_REQUESTED** [review]
The code looks very clean and well organised. However, the extra string appended to the LOCAL_STORAGE_ERROR_SAVE_DATA variable in Messages.java could cause the test cases to fail if the LocalStorage class was to throw an error. Could you remove the extra string? Thank you!

File: src/main/java/seedu/moneygowhere/common/Messages.java (hunk 1 above)
Comment (penguin-s) [reply src/main/java/seedu/moneygowhere/common/Messages.java:54]: Oh good catch, thank you for pointing it out!

**Review by xzynos — APPROVED** [review]
Thank you very much for your prompt changes!

**Pull request merged**